development-tools/clinic.rst

```diff
@@ -2001,6 +2001,54 @@ The generated glue code looks like this:
 .. versionadded:: 3.13
 
 
+.. _clinic-howto-getter:
+
+How to generate a getter
+------------------------
+
+You can use the ``@getter`` directive to generate an "impl" function
+that defines a `getter <https://docs.python.org/3/c-api/structures.html#c.PyGetSetDef>`_.
+
+This example --- taken from :cpy-file:`Modules/_io/bufferedio.c` ---
+shows the use of ``@getter`` in combination with
+the :ref:`@critical_section <clinic-howto-critical-sections>` directive
+(which achieves thread safety without causing deadlocks between threads)::
+
+    /*[clinic input]
+    @critical_section
+    @getter
+    _io._Buffered.closed
+    [clinic start generated code]*/
+
+The generated glue code looks like this:
+
+.. code-block:: c
+
+    static PyObject *
+    _io__Buffered_closed_get(buffered *self, void *context)
+    {
+        PyObject *return_value = NULL;
+
+        Py_BEGIN_CRITICAL_SECTION(self);
+        return_value = _io__Buffered_closed_get_impl(self);
+        Py_END_CRITICAL_SECTION();
+
+        return return_value;
+    }
+
+And then the implementation will work the same as the Python method which is
+decorated by `property <https://docs.python.org/3/library/functions.html#property>`_.
+
+.. code-block:: python
```

Comment:
REPL examples should use `pycon`, which stands for "Python console", and instructs Sphinx to apply syntax highlighting tailored for the Python REPL

```suggestion
.. code-block:: pycon
```

Reply:
Oh not PyCon.. lol

Reply:
I absolutely thought it was a typo the first time I saw `.. code-block:: pycon` 😆